docs/usage/syncing-instances.md

```diff
@@ -2,11 +2,60 @@
 sidebar_position: 4
 ---
 # Syncing Instances
-So, how to sync every instance of HuronOS at the same time?, you might ask. The answer is: using a directives file. Where to retrieve this file from is defined when a HuronOS usb is created, and that file is retrieved every minute for every HuronOS instance (although that might not happen at the same time) so, if the directives file url is correctly configured and set on the usbs, every instance should be in sync with the directives file wherever that file is stored.
 
+**Remote syncing** is one of the flagship features of huronOS, allowing you to synchronize preferences, configurations, and behavior directives with **several huronOS instances** (running installations of huronOS) using something we call *the directives file*. This file is a plain text file with a specific syntax known as *huronOS Directives File Syntax*, resulting in a *.hdf* file.
 
-# Troubleshoot
-### My directives do not sync
-- Be sure your file is accessible from other machines.
-- Verify the instance you're trying to sync has an internet connection available.
-- Verify that your expected directives file reflects the one stored at your server.
\ No newline at end of file
+To sync an instance, you need to specify a directives file URL that can be accessed via HTTP or HTTPS. This URL can be a domain name or an IP address. The huronOS **sync manager** polls this file every minute, comparing it to the local version. If any changes are detected, the sync manager processes them and triggers the necessary updates to align the system with the new rules. Be aware you can expect up to 2 minutes from the edition to the full propation.
+
+To prevent firewall issues, the IP address of the specified directives file URL is added to the exceptions list when the firewall is set to *DROP* mode. If the directives URL contains a domain name instead of an IP address, the system relies on the DNS system to resolve the domain to all IP addresses hosting that domain.
```

Comment:
To prevent firewall issues, the IP address belonging to (or it could be "affiliated to", "attached to"; this specific change is not so necessary but I think it could help make it a little bit more clear) the specified directives file's URL is added to the [...]

Reply:
> To prevent firewall issues, the IP address belonging to (or it could be "affiliated to", "attached to"; this specific change is not so necessary but I think it could help make it a little bit more clear) the specified directives file's URL is added to the [...]

Not sure about this @alex-gorod, in this case the URL instead of looking like "https://google.com/somefile.hdf" looks like "https://192.168.1.20/somefile.hdf", the replacement of the *domain name* is an IP address, so in this case I don't think belong/affiliated/attached describes correctly that the IP contained on the URL is the one to be added as an exception to the firewall. What do you suggest then?

Reply:
then you could leave it as it was :) that's why I mentioned the change wasn't as necessary.